sage_codes/curve_derivation

```diff
@@ -0,0 +1,37 @@
+sage: q = 2^252 + 27742317777372353535851937790883648493                    
```

Comment:
Please, reference on this file in a comment or whatever you prefer that it was taken from:
https://github.com/zkcrypto/jubjub/blob/master/doc/derive/derive.sage and then modified.

Credit the authors also.

Reply:
Okay, i’ll add this in a subsection of the derivation file and a written explanation of where it came from. 
Additionally, this is going to need us to discuss and explain the choice of this prime, as the scalar field of curve25519, in terms of DL based proofs etc. So more references are necessary.